wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/wsplugins/PluginFQNParserTest.java

```diff
@@ -50,6 +53,16 @@ public void setUp() throws Exception {
     parser = new PluginFQNParser();
   }
 
+  @Test
+  public void test() {
+    Lifecycle lifecycle = new Lifecycle();
+    lifecycle
+        .postStart(new Handler().exec(new Exec().command(Collections.singletonList("test"))))
+        .preStop(new Handler().exec(new Exec().command(Collections.singletonList("test"))));
+
+    System.out.println(lifecycle.toString());
```

Comment:
This is not testing anything.

Reply:
yup, thanks, commit it by mistake

Reply:
would be great to add some coverage for the new lifecycle handlers

Reply:
fixed